docs/data-sources/dns_filtering_profile.md

```diff
@@ -0,0 +1,100 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_dns_filtering_profile Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  DNS filtering gives you the ability to control what websites your users can access. For more information, see Twingate's documentation https://www.twingate.com/docs/dns-filtering.
+---
+
+# twingate_dns_filtering_profile (Data Source)
+
+DNS filtering gives you the ability to control what websites your users can access. For more information, see Twingate's [documentation](https://www.twingate.com/docs/dns-filtering).
```

Comment:
Prior to "For more information...", I would add the sentence "DNS filtering is only available on certain plans."